data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -105,8 +105,7 @@ t_reviewed_preprints_with_under_review_process AS (
       biorxiv_medrxiv_normalized_title)
   FROM `elife-data-pipeline.prod.v_manuscript_with_matching_preprint_server_doi`
   WHERE preprint_doi IS NOT NULL
-  AND is_or_was_under_review -- to include the "six" additional manuscripts
-  AND long_manuscript_identifier NOT LIKE '%VOR%'
+  AND provide_docmaps_for
```

Comment:
Move `provide_docmaps_for` here, or use a term more related to the business (but we couldn't come up with a good name).

If we keep a boolean, suggest to start with is, was or should.